docs/external_cmps.md

```diff
@@ -0,0 +1,81 @@
+# External components
+
+By default, plain ```docker-compose up```, regardless of the
+pipeline will start containers for following services:
```

Comment:
pipeline**,** will start containers for **the** following

Reply:
Done